tests/test_file_registry.py

```diff
@@ -31,9 +31,11 @@ def test_init_from_existing_index(populated_index_dir):
     r.get("name", "version")
 
 
-def test__make_id():
+def test_make_id():
```

Comment:
Ahah that's my OCD strictly following the convention `test_<func_name>` but it's ok 👍